product_brand/readme/CREDITS.md

```diff
@@ -1 +1,2 @@
 The migration of this module from 16.0 to 17.0 was financially supported by Camptocamp.
```

Comment:
You can remove the first sentence regarding 16.0 to 17.0 migration, and keep only the new one.

Reply:
I have updated it to give the most logical output.